### PR TITLE
Badges badges badges

### PR DESCRIPTION
**Pull request by astorije**
Pretty pretty badges, because red looks nicer in SVG!

### CODE REVIEW

**Comment by floogulinc** [comment]
Your `LICENSE` link is broken because there is no `LICENSE` file.

**Comment by astorije** [comment]
Ah, well spotted! Adding one right now!

**Comment by floogulinc** [comment]
You might want to change the name on the copyright to `Mattias Erming` before it is merged.

**Comment by astorije** [comment]
-__-'
Sorry, another paste hastily done...
Amended my last commit, thanks for your thorough eyes!

**Comment by astorije** [comment]
@JocelynDelalande, @erming, may I have your respective :+1: on this?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Mattias Erming
```
Comment (JocelynDelalande) [inline LICENSE]: add « and contributors », since there is no [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) as far as I know.

Comment (astorije) [inline LICENSE]: Oh boy, that moment at the end of a talk when you hope no one would raise their hands when you ask "Any questions?"

I don't know. This is a tricky question. At first I was like, sure why not, then after reading [this](http://crypti.cc/blog/2015/02/17/contributor-license-agreement/), I'm like, heck, that's why not. Honestly, we **do not** want a CLA, that would just get in the way of enhancing that project (I know of one project I contribute to that requires to sign a CLA, and boy it's tedious for everyone).

This seems like we would need a keen eye on this. I'm sure she is very busy, but if @wseltzer happens to receive notifications, and would be okay to help us a bit, that would be great :-)
@wseltzer, essentially, this is an Free and Open Source Software project under the MIT license and we get *a lot* of contributions. What would be the best thing to type here? Of course it's always nice when we give the right credit to the right people, but on the other hand if that forces us to get agreement from all contributors if we decide to change license), that would be hard to organize.

@JocelynDelalande, what I am sure about though, is that I want this PR to make it into master quickly :-) Can I get your +1 *and* continue the discussion to reach to a solution even if it has to be done post-merged, in another PR?

Comment (JocelynDelalande) [inline LICENSE]: > @JocelynDelalande, what I am sure about though, is that I want this PR to make it into master quickly :-) Can I get your +1 and continue the discussion to reach to a solution even if it has to be done post-merged, in another PR?

Sure :) so please reflect current project state : copyright holder is "Mathias Erming and contributors", I was not discussing if we wanted a CLA or not, that's outside the scope of this PR.

Comment (astorije) [inline LICENSE]: Aaah yeah, sure, whatever, done in 92d2466.

**Comment by erming** [comment]
-1 for me. Sure, you can add the license if you want (even though I don't think it's needed). But I don't know about the badges. I just think they're ugly and unnecessary.

**Comment by astorije** [comment]
@erming, badges are a nice display of seriousness accessible on a first glance.
Almost all open source projects out there provide a Travis CI badge for example, among other things to assess overall stability, and dependency badges are also very useful when assessing security concerns (deps flagged as insecure).

Let's follow well accepted/de facto open source conventions here please.

Also, dude, it's *SVG badges*, how can you find them ugly! :D (PNG badge on the current README weeps my eyes on Retina ^^)

**Comment by astorije** [comment]
Also, it adds transparency (if our project is broken, we don't hide it), which is always a good thing.

**Comment by williamboman** [comment]
+1 For badges, definitely a good addition.

**Comment by JocelynDelalande** [comment]
That's a :+1: for me also.

@erming those badges are not a matter of being « flashy & shiny », (yes, visual aspect is debatable but it's bikeshedding IMHO).

The matter is : it allows some visitor to evaluate quickly your project : is it active ? are dependencies up-to-date ? are tests passing ?

I'm evaluating OSS projects on a daily basis, the information carried by badges is valuable as hints about  software quality.

It's not about having a badges collection, it's about being transparent on software quality.

Another point in favor of those badges is: if they are turning red, it will give us a reminder to take action in order to get them green.

**Comment by JocelynDelalande** [comment]
@astorije  is PR title in link with http://www.badgerbadgerbadger.com/ ?

**Comment by astorije** [comment]
> @astorije is PR title in link with http://www.badgerbadgerbadger.com/ ?

What else? ^^

Also thanks @JocelynDelalande for putting better words on my exact thoughts. Hope that @erming will change his mind :-)

**Comment by astorije** [comment]
Ah, another pro-badges argument: I froze for a moment where trying to get to our Travis CI builds. "Wait, where do I click already? Right, I need to manually type the URL..." :-)

**Comment by Calinou** [comment]
:+1:

**Comment by astorije** [comment]
Rebased on master to comply with changes added by #508.

**Comment by erming** [comment]
Remove `License` and `devDependencies` and it's all good!

**Comment by astorije** [comment]
Whyyyyyy, please nooooo! What's wrong with clearly expressing what free license we are using and easily linking to it?!

**Comment by astorije** [comment]
I live to fight another day. Removed, merging now.

**Pull request merged**